packages/story-editor/src/components/panels/design/captions/captions.js

```diff
@@ -96,20 +96,19 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
 
   const handleChangeTrack = useCallback(
     ({ src = '', id, needsProxy = false }) => {
-      const newTracks = {
+      const newTrack = {
         track: src,
         trackId: id,
-        trackName: src.split('/').pop(),
```

Comment:
Why is this being removed?

Reply:
There is no need to store `trackName` in story state, because this can simply be done during render.

This was moved to here:

https://github.com/GoogleForCreators/web-stories-wp/blob/a34a95b4edf810e62f2e800886ebce5dfa3ddbc6/packages/story-editor/src/components/panels/shared/media/captionsPanelContent.js#L134